### ISSUE TITLE
Modify Auth Password-Policy

### ISSUE THREAD

**Issue by Mikej81** [F5Networks/f5-declarative-onboarding#122]
Need to be able to modify Auth Password-Policy.  This is tied to several industry compliance guidelines.

Primary auth (for *MY* use-case) will always be off-box auth, but local user password policy should be enforced for credentials of last resort, i.e., Local Admin if AD/Radius down.

Example...

tmsh modify auth password-policy expiration-warning 7
tmsh modify auth password-policy max-duration 90
tmsh modify auth password-policy max-login-failures 3
tmsh modify auth password-policy min-duration 1
tmsh modify auth password-policy minimum-length 8
tmsh modify auth password-policy password-memory 3
tmsh modify auth password-policy policy-enforcement enabled
tmsh modify auth password-policy required-lowercase 2
tmsh modify auth password-policy required-numeric 2
tmsh modify auth password-policy required-special 2
tmsh modify auth password-policy required-uppercase 2

```json
{
...
  "passpolicy": {
    "class": "passwordPolicy",
     "expiration-warning": "7",
     ...
   }

}
```

**Comment by Mikej81** [comment]
Also need 
tmsh modify auth remote-user

**Comment by dstokesf5** [comment]
I have added a feature request to modify password policy settings to our internal product backlog as AUTOTOOL-1379.

It looks like remote user settings can already be configured with DO: https://clouddocs.f5.com/products/extensions/f5-declarative-onboarding/latest/bigip-examples.html?highlight=remote%20user#configuring-big-ip-authentication-methods

**Comment by seattlevine** [comment]
Password policy was implemented as part of #316. 

Remote user has been available as part of the Authentication class for quite a while. Let us know in a new issue if you need something different there.

Closing as a duplicate (even though this one was first)

**Comment by seattlevine** [comment]
Duplicate of #316
